python/examples/01_tensorflow-basic.ipynb

```diff
@@ -0,0 +1,290 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# TensorFlow model serving with Konduit Serving\n",
+    "\n",
+    "This document illustrates a simple client-server interaction to perform inference on a TensorFlow model using the Python SDK for Konduit Serving.  \n",
+    "\n",
+    "This tutorial is split into two parts: \n",
+    "\n",
+    "1. Configuration \n",
+    "2. Running the server"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from konduit import ParallelInferenceConfig, ServingConfig, TensorFlowConfig, ModelConfigType\n",
+    "from konduit import TensorDataTypesConfig, ModelPipelineStep, InferenceConfiguration\n",
+    "from konduit.server import Server\n",
+    "from konduit.client import Client\n",
+    "\n",
+    "import time\n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Configuration\n",
+    "\n",
+    "Konduit Serving works by defining a series of \"pipeline steps\". These include operations such as \n",
+    "1. Pre- or post-processing steps\n",
+    "2. One or more machine learning models\n",
+    "3. Transforming the output in a way that can be understood by humans\n",
+    "\n",
+    "If deploying your model does not require pre- nor post-processing, only one pipeline step - a machine learning model - is required. This configuration is defined using a single `ModelPipelineStep`. \n",
+    "\n",
+    "Before running this notebook, you should run the `build_jar.py` script and copy the JAR (`konduit.jar`) to this folder. \n",
+    "\n",
+    "Start by downloading the model weights to the `data` folder. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!curl https://deeplearning4jblob.blob.core.windows.net/testresources/bert_mrpc_frozen_v1.zip --output ../data/bert.zip\n",
+    "!unzip ../data/bert.zip "
```

Comment:
Can you mention OS specific commands so these can run inside Windows, MacOS and Linux. If they need to install anything then you can mention that as well. A good way to handle some items is to write the download and extration script in python

Reply:
Replaced as follows: 

```python 


from urllib.request import urlretrieve 
from zipfile import ZipFile
urlretrieve("https://deeplearning4jblob.blob.core.windows.net/testresources/bert_mrpc_frozen_v1.zip", "../data/bert.zip")
with ZipFile('../data/bert.zip', 'r') as zipObj:
    zipObj.extractall()
```